pykickstart/commands/ostreesetup.py

```diff
@@ -100,3 +101,8 @@ class F38_OSTreeSetup(F21_OSTreeSetup):
     removedKeywords = KickstartCommand.removedKeywords
     removedAttrs = KickstartCommand.removedAttrs
     conflictingCommands = ["ostreecontainer"]
+
+class F43_OSTreeSetup(F38_OSTreeSetup):
+    removedKeywords = KickstartCommand.removedKeywords
+    removedAttrs = KickstartCommand.removedAttrs
+    conflictingCommands = ["ostreecontainer", "bootc"]
```

Comment:
Please add bootc to the ostreesetup conflict test

Reply:
Done